vg/font.go

```diff
@@ -105,16 +111,20 @@ func (f *Font) Name() string {
 	return f.name
 }
 
-// Font returns the corresponding truetype.Font.
-func (f *Font) Font() *truetype.Font {
+// Font returns the corresponding opentype.Font.
+func (f *Font) Font() *opentype.Font {
 	return f.font
 }
 
 func (f *Font) FontFace(dpi float64) font.Face {
-	return truetype.NewFace(f.font, &truetype.Options{
+	face, err := opentype.NewFace(f.font, &opentype.FaceOptions{
 		Size: f.Size.Points(),
 		DPI:  dpi,
 	})
+	if err != nil {
+		panic(err)
+	}
+	return face
 }
```

Comment:
We need to consider whether this needs to be altered to return the error. At the moment `opentype.NewFace` only ever returns a nil `error`, but we shouldn't panic here in normal use probably.